tests/Vendor/Order/OrderServiceTest.php

```diff
@@ -334,6 +334,18 @@ public function testGetOrdersAmountsTaxesDetails(): void
         }
     }
 
+    public function testSetDetailsOnAnOrder(): void
+    {
+        $orderService = $this->buildVendorOrderService();
+
+        $details = "Voici un nouveau commentaire de vendeur sur la commande";
+        $orderService->setOrderDetails(5, $details);
+
+        $order = $orderService->getOrderById(5);
+
+        static::assertSame($details, $order->getDetails());
```

Comment:
```suggestion
        static::assertSame($details, $orderService->getOrderById(5)->getDetails());
```
Comme ça pas de variable $order